targeteditor/src/main/scala/explore/targeteditor/TargetBody.scala

```diff
@@ -38,7 +37,9 @@ final case class TargetBody(
 }
 
 object TargetBody extends ModelOptics {
-  type Props = TargetBody
+  type Props          = TargetBody
+  // search term, after, on empty, on error
+  type SearchCallback = (NonEmptyString, Callback, Callback, Throwable => Callback) => Callback
```

Comment:
Can we use a case class for this?

Reply:
Yes, it maybe better to do that and get better names for the params